src/App.js

```diff
@@ -1,9 +1,11 @@
 import "./assets/style/app.scss";
 import { Route, Routes } from "react-router-dom";
 import Home from "./pages/Home/Home";
+import Navbar from "./pages/Shared/Navbar";
 function App() {
   return (
     <div>
+      <Navbar></Navbar>
```

Comment:
Is there any reason for writing component with closing tag ? We can write component with self closing tag.
```suggestion
      <Navbar />
```